inception-project/inception
# #4562 - Project template for named entity annotation

**What's in the PR**
- [x] Added new entity annotation template
- [x] New thumbnail 

**How to test manually**
* Try creating a project with the new template and check the layer and recommender configurations

**Automatic testing**

**Documentation**
